Automattic/sensei
# Fix spacing issues on Emails settings page

This PR wraps the title, links and tabs of the Emails settings page in the `sensei-custom-navigation` element so that it matches the markup of other Sensei pages (Courses, Lessons, etc.). Doing this also adjusts the spacing so that it looks similar to those other pages. Note that this may not match exactly with the spacing in the designs, but I think consistency between Sensei pages is more important here. Or, if we end up tweaking the spacing here, it should affect the other pages as well because of the shared markup.

The language that is being used for some of the elements is a bit confusing. I'm thinking of the General | Appearance | Courses etc. options as links (since they're inside the `.sensei-custom-navigation__links` element), and the Student Emails | Teacher Emails | Settings options as tabs (since they're inside the `.sensei-custom-navigation__tabbar` element).

### New Action
`sensei_settings_after_links` - Fires after the navigation links are displayed.

### Updated Filter
`sensei_settings_content` - Filters content for the settings page.

### Testing instructions
Navigate to the Emails settings page and ensure it looks similar to the screen shot:

![Screenshot 2023-02-13 at 3 51 23 PM](https://user-images.githubusercontent.com/1190420/218571832-0695302e-b9db-4f26-b8d6-2d958761d19b.jpg)
